bot/exts/backend/error_handler.py

```diff
@@ -0,0 +1,157 @@
+        """Build a basic embed with red colour and either a random error title or a title provided."""
+        embed = Embed(colour=Colours.soft_red)
+        if isinstance(title, str):
+            embed.title = title
+        else:
+            embed.title = random.choice(title)
+        embed.description = message
+        return embed
+
+    @commands.Cog.listener()
+    async def on_command_error(
+        self, ctx: commands.Context, error: commands.CommandError
+    ) -> None:
+        """Activates when a command opens an error."""
+        if getattr(error, "handled", False):
+            logger.debug(
+                f"Command {ctx.command} had its error already handled locally; ignoring."
+            )
+            return
+
+        error = getattr(error, "original", error)
+        logger.debug(
+            f"Error Encountered: {type(error).__name__} - {str(error)}, "
+            f"Command: {ctx.command}, "
+            f"Author: {ctx.author}, "
+            f"Channel: {ctx.channel}"
+        )
+
+        if isinstance(error, commands.CommandNotFound):
+            return
+
+        elif isinstance(error, commands.UserInputError):
+            self.revert_cooldown_counter(ctx.command, ctx.message)
+            embed = self.error_embed(
+                f"Your input was invalid: {error}\n\nUsage:\n"
+                f"```{ctx.prefix}{ctx.command} {ctx.command.signature}```"
+            )
+            await ctx.send(embed=embed)
+            return
+
+        elif isinstance(error, commands.CommandOnCooldown):
+            mins, secs = divmod(math.ceil(error.retry_after), 60)
+            embed = self.error_embed(
+                f"This command is on cooldown:\nPlease retry in **{mins} minutes {secs} seconds**.",
+                NEGATIVE_REPLIES,
+            )
+            await ctx.send(embed=embed, delete_after=10)
+            return
+
+        elif isinstance(error, commands.DisabledCommand):
+            await ctx.send(
+                embed=self.error_embed(
+                    "This command has been disabled.", NEGATIVE_REPLIES
+                )
+            )
+            return
+
+        elif isinstance(error, commands.NoPrivateMessage):
+            await ctx.send(
+                embed=self.error_embed(
+                    "This command can only be used in the server.", NEGATIVE_REPLIES
+                )
+            )
+            return
+
+        elif isinstance(error, commands.BadArgument):
+            self.revert_cooldown_counter(ctx.command, ctx.message)
+            embed = self.error_embed(
+                "The argument you provided was invalid: "
+                f"{error}\n\nUsage:\n```{ctx.prefix}{ctx.command} {ctx.command.signature}```"
+            )
+            await ctx.send(embed=embed)
+            return
+        else:
+            await self.handle_unexpected_error(ctx, error)
+            return  # Exit early to avoid logging.
```

Comment:
```suggestion
            return  # Exit early to avoid logging.

        logger.debug(
            f"Error Encountered: {type(error).__name__} - {str(error)}, "
            f"Command: {ctx.command}, "
            f"Author: {ctx.author}, "
            f"Channel: {ctx.channel}"
        )
```
Logging can be done ere to ensure that unhandled errors are not logged twice.